woocommerce/woocommerce-ios
# Dark Mode: update colors for Review details

Reviews for #1555 

## Changes

- Removed some unused code I noticed when finding the Review cells:
  - `NoteDetailsRow.swift`
  - `NoteDetailsHeaderTableViewCell.swift`
  - `NoteDetailsHeaderTableViewCell.xib`
- Updated colors Review details comment table view cell `NoteDetailsCommentTableViewCell` with some `UIButton` extension helpers

## Testing

- Launch the app
- Go to Reviews tab
- Tap on a review --> the action buttons should have the expected colors

## Example screenshots

Light mode | Dark mode
-- | --
![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 17 18 19](https://user-images.githubusercontent.com/1945542/70311939-8ddf6e00-184d-11ea-997e-2ea13fdfbdf0.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-06 at 17 18 34](https://user-images.githubusercontent.com/1945542/70311942-8ddf6e00-184d-11ea-95ef-66ffafb08a05.png)


Update release notes:
